airflow/models/dagbag.py

```diff
@@ -269,7 +270,12 @@ def _load_modules_from_file(self, filepath, safe_mode):
                 return [new_module]
             except Exception as e:  # pylint: disable=broad-except
                 self.log.exception("Failed to import: %s", filepath)
-                self.import_errors[filepath] = str(e)
+                if conf.getboolean('core', 'dagbag_import_error_tracebacks'):
```

Comment:
Should we add `fallback` just in case someone don't have this option in config?

Reply:
Won't it just use the default in that case?

Reply:
https://github.com/apache/airflow/blob/master/airflow/configuration.py#L334

Reply:
Only if
https://github.com/apache/airflow/blob/ce66bc944d246aa3b51cce6e2fc13cd25da08d6e/airflow/configuration.py#L338

So if users will not update their config they will face an unknown error

Reply:
`self.airflow_defaults` is a parsed version of `default_airflow.cfg`, so I don't think we need to double up with a fallback here. They won't run into errors if they don't add stuff to their `airflow.cfg`.

Reply:
Yup.